ai-eval-service/main.py

```
@@ -76,9 +76,42 @@ def run_worker(payload: dict, timeout: int = 300):
             check=False
         )
         
+        stdout_str = result.stdout.strip()
+        def parse_stdout_json(text: str) -> dict:
+            decoder = json.JSONDecoder()
+
+            def is_valid_schema(obj: Any) -> bool:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `obj`

(ANN401)